Package.swift

```diff
@@ -8,9 +8,15 @@ let package = Package(
   platforms: [.macOS(.v10_15)],
   dependencies: [
     .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.1.1"),
-    .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "0.2.0")),
-    .package(url: "https://github.com/apple/swift-tools-support-core.git", .upToNextMinor(from: "0.1.3")),
-    .package(url: "https://github.com/OpenCombine/OpenCombine.git", .upToNextMinor(from: "0.10.0")),
+    .package(
+      url: "https://github.com/apple/swift-argument-parser", 
```

Comment:
<!--
  1 failure:  Lines should not ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Package.swift;
  Line: 12;
-->
- :no_entry_sign: Lines should not have trailing whitespace. (`trailing_whitespace`)